extensions/wikia/AdEngine/js/provider/gpt/sourcePointTag.js

```diff
@@ -5,17 +5,54 @@ define('ext.wikia.adEngine.provider.gpt.sourcePointTag', [
 	'ext.wikia.adEngine.adTracker',
 	'ext.wikia.adEngine.provider.gpt.googleTag',
 	'wikia.document',
+	'wikia.lazyqueue',
 	'wikia.log',
 	'wikia.window'
-], function (adContext, adTracker, GoogleTag, doc, log, window) {
+], function (adContext, adTracker, GoogleTag, doc, lazyQueue, log, window) {
 	'use strict';
 
-	var logGroup = 'ext.wikia.adEngine.provider.gpt.sourcePointTag',
-		sourcePointClientId = 'rMbenHBwnMyAMhR',
-		context = adContext.getContext();
+	var blocking = false,
+		context = adContext.getContext(),
+		logGroup = 'ext.wikia.adEngine.provider.gpt.sourcePointTag',
+		sourcePointClientId = 'rMbenHBwnMyAMhR';
+
+	/**
+	 * Create cssText based on computedStyle
+	 * Custom method because of Firefox bug #137687
+	 */
+	function getComputedStyleCssText(element) {
+		var style = window.getComputedStyle(element),
+			cssText;
+
+		if (style.cssText !== '') {
+			return style.cssText;
+		}
+
+		cssText = '';
+		for (var i = 0; i < style.length; i++) {
+			cssText += style[i] + ':' + style.getPropertyValue(style[i]) + ';';
+		}
+
+		return cssText;
+	}
+
+	function copyStyles(from, to) {
```

Comment:
Those two functions look like things which don't really belong to `SourcePointTag`. Shall we create `ext.wikia.adEngine.cssTweaker`?